ext/RMagick/extconf.rb

```diff
@@ -308,9 +308,7 @@ def assert_can_compile!
     end
 
     def assert_minimum_ruby_version!
-      unless checking_for("Ruby version >= #{MIN_RUBY_VERS}") do
-        Gem::Version.new(RUBY_VERSION) >= Gem::Version.new(MIN_RUBY_VERS)
-      end
+      unless checking_for("Ruby version >= #{MIN_RUBY_VERS}") { Gem::Version.new(RUBY_VERSION) >= Gem::Version.new(MIN_RUBY_VERS) }
```

Comment:
@Watson1978 I find this more readable wrapped to the next line.

Reply:
hmm, I dislike same indentation of `end`... 
```ruby
      unless checking_for("Ruby version >= #{MIN_RUBY_VERS}") {}
        Gem::Version.new(RUBY_VERSION) >= Gem::Version.new(MIN_RUBY_VERS)
      end
        exit_failure "Can't install RMagick #{RMAGICK_VERS}. Ruby #{MIN_RUBY_VERS} or later required.\n"
      end
```

ok I need more thinking time.

Reply:
Yeah, this syntax is kind of confusing, now that I look at it again. Can we do:

```rb
checking_for("Ruby version >= #{MIN_RUBY_VERS}")

return if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new(MIN_RUBY_VERS)

exit_failure "Can't install RMagick #{RMAGICK_VERS}. Ruby #{MIN_RUBY_VERS} or later required.\n"
```